src/Tribe/Admin/Screen_Options/Attendees.php

```diff
@@ -62,7 +62,8 @@ protected function add_column_headers_options() {
 	 * @return array
 	 */
 	public function filter_manage_columns( array $columns ) {
-		return Tribe__Tickets__Attendees_Table::get_table_columns();
+		$attendees_table = new Tribe__Tickets__Attendees_Table();
```

Comment:
We should take this moment to register the class with tribe() where we register the others